README.md

```diff
@@ -1635,6 +1659,267 @@ map.addEntryListener(mapEventListener).then(function () {
+client.getMap('persons').then(function (mp) {
+    personMap = mp;
+    return personMap.put('Ahmet', 28);
+}).then(function () {
+    return personMap.put('Ali', 30);
+}).then(function () {
+    return personMap.put('Furkan', 23);
+}).then(function () {
+    var predicate = new Predicates.sql('__key like F%');
+    return personMap.valuesWithPredicate(predicate);
+}).then(function (startingWithA) {
+    console.log(startingWithA.get(0)); // 23
+});
+```
+
+In this example, the code creates a list with the values whose keys start with the letter "F”.
+
+You can use `this` attribute to perform a predicated search for entry values. Please see the following example:
+
+```javascript
+var personMap;
+client.getMap('persons').then(function (mp) {
+    personMap = mp;
+    return personMap.put('Ahmet', 28);
+}).then(function () {
+    return personMap.put('Ali', 30);
+}).then(function () {
+    return personMap.put('Furkan', 23);
+}).then(function () {
+    var predicate = new Predicates.greaterEqual('this', 27);
+    return personMap.valuesWithPredicate(predicate);
+}).then(function (olderThan27) {
+    console.log(olderThan27.get(0), olderThan27.get(1)); // 28 30
+});
+```
+
+In this example, the code creates a list with the values greater than or equal to "27".
+
+#### 7.7.1.4. Filtering with Paging Predicates
+
+The Node.js client provides paging for defined predicates. With its `PagingPredicate` object, you can get a list of keys, values, or entries page by page by filtering them with predicates and giving the size of the pages. Also, you can sort the entries by specifying comparators.
+
+```javascript
+var map;
+hazelcastClient.getMap('students').then(function (mp) {
+    map = mp;
+
+    var greaterEqual = Predicates.greaterEqual('age', 18);
+    var pagingPredicate = Predicates.paging(greaterEqual, 5);
+
+// Set page to retrieve third page
+    pagingPredicate.setPage(3);
+
+    // Retrieve third page
+    return map.valuesWithPredicate(pagingPredicate)
+}).then(function (values) {
+    // some operations
+...
+
+    // Set up next page
+    pagingPredicate.nextPage();
+
+    // Retrieve next page
+    return map.valuesWithPredicate(pagingPredicate);
+}).then(function (values) {
+    // some operations
+});
+```
+
+If you want to sort the result before paging, you need to specify a comparator object that implements `Comparator` interface. Also, this comparator object should be one of `IdentifiedDataSerializable` or `Portable`. After implementing the Node.js version, you need to implement the Java equivalent of the comparator and its factory. The Java equivalent of the comparator should implement `java.util.Comparator`. Note that `comprare` function of `Comparator` on the Java side is the equivalent of the `sort` function of `Comparator` on the Node.js side. When you implement `Comparator` and its factory, you can add them to the `CLASSPATH` of the server side.  See [Adding User Library to CLASSPATH section](#adding-user-library-to-classpath).
```

Comment:
... implements the `Comparator` interface...

... Note that the `compare` function of `Comparator` on ...
(note the typo "comprare" is fixed here too)